x-pack/plugin/monitoring/src/main/resources/monitoring/watches/elasticsearch_nodes.json

```diff
@@ -158,7 +158,6 @@
     "add_to_alerts_index": {
       "index": {
         "index": ".monitoring-alerts-6",
```

Comment:
`,` needs to be removed, otherwise this file can't be parsed

Reply:
Incidentally, @pickypg, do you recall why this particular watch doesn't set a document ID whereas all the other cluster alerts watches do?

Reply:
@ycombinator It was briefly before the Task Manager stuff began in Kibana (and partly what led to it), but because you could restart multiple nodes in sequence and you didn't want to lose the history of all of them. The goal was if this index ever became a burden to just delete it and recreate it (anecdotally I've never seen the index with >500 documents) or move to index rollover.